shell/platform/windows/angle_surface_manager.cc

```diff
@@ -343,6 +343,13 @@ void AngleSurfaceManager::SetVSyncEnabled(bool enabled) {
     LogEglError("Unable to update the swap interval");
     return;
   }
+
+  if (eglMakeCurrent(egl_display_, EGL_NO_SURFACE, EGL_NO_SURFACE,
+                     EGL_NO_CONTEXT) != EGL_TRUE) {
+    LogEglError(
+        "Unable to release the context after updating the swap interval");
+    return;
+  }
```

Comment:
This is what fixes the ANGLE roll. See: https://github.com/flutter/engine/pull/45064

This is covered by existing tests and _should_ be causing failures as it causes the context to be current on multiple threads.